core21/Access.Odbc.LinqService/Tests/Linq/ParameterTests/Tests.Linq.ParameterTests.Issue404Test(Access.Odbc.LinqService).sql

```diff
@@ -187,21 +187,16 @@ SELECT
 FROM
 	(
 		SELECT DISTINCT
-			[t1].[Id]
+			[c_1].[Id]
 		FROM
-			(
-				SELECT TOP 1
-					[c_1].[Id]
-				FROM
-					[Table404One] [c_1]
-			) [t1]
+			[Table404One] [c_1]
 	) [key_data_result]
 		INNER JOIN [Table404Two] [_v] ON ([_v].[FirstTableId] = [key_data_result].[Id])
 
 BeforeExecute
 -- Access.Odbc AccessODBC
 
-SELECT TOP 1
```

Comment:
test sql changed, but it is fine as it doesn't affect tested functionality https://github.com/linq2db/linq2db/issues/404